varcode/read_evidence/pileup_collection.py

```diff
@@ -585,3 +591,22 @@ def from_bam(pysam_samfile, loci):
             if close_on_completion:
                 pysam_samfile.close()
 
+def to_locus(variant_or_locus):
+    """
+    Return a Locus object for a Variant instance.
+
+    Since the read evidence module uses a different Variant class than the rest
```

Comment:
Maybe mention where someone can go to see the definition of this different Variant class?